### PR TITLE
Story2

### PR DESCRIPTION
**Pull request by jasonrees1**

### CODE REVIEW

File: function.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        $seriesHtml .= '<p>End Date: ' . $serie['end_date'] . '</p>';
+        $seriesHtml .= '<p>Number Of Seasons: ' . $serie['number_of_seasons'] . '</p>';
+        $seriesHtml .= '<p>Number Of Episodes: ' . $serie['number_of_episodes'] . '</p>';
+        $seriesHtml .= '<p>Director: ' . $serie['director'] . '</p>';
+        $seriesHtml .='</div>';
+    }
+    return $seriesHtml;
+}
+
+
```
Comment (sjr-mayden) [inline function.php]: Unused newlines here

File: index.php [hunk 2]
```diff
@@ -5,21 +5,38 @@
     <title>Netflix Series Collection</title>
 </head>
 <body>
+<h2>Complete all fields to add a new series to your collection</h2>
+<!-- -->
```
Comment (sjr-mayden) [inline index.php]: This comment line can be deleted

File: index.php [hunk 3]
```diff
@@ -5,21 +5,38 @@
...
+    Director:
+    <input type="text" name="director">
+    <input type="submit" value="submit">
+</form>
+
 <?php
-$db = new PDO('mysql:host=db;dbname=netflix_series','root','password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
-$query = $db-> query('SELECT `id`,`title`,`start_date`,`end_date`,`number_of_seasons`,`number_of_episodes`,`director` FROM `favourites`');
+//As function connect_to_netflix_series_db has no parameters it will run here
```
Comment (sjr-mayden) [inline index.php]: Fine for your notes but something to get in the habit of removing from production code

File: function.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<?php
```
Comment (sjr-mayden) [inline function.php]: Rename file to `functions.php` because it holds more than one function

File: index.php [hunk 5]
```diff
@@ -5,21 +5,38 @@
     <title>Netflix Series Collection</title>
 </head>
 <body>
+<h2>Complete all fields to add a new series to your collection</h2>
+<!-- -->
+<form method="post" action="adddata.php">
+    Series Title:
+    <input type="text" name="seriestitle">
```
Comment (sjr-mayden) [inline index.php]: Can use the `required` attribute for these input fields (step 1 of the 4 that we talked about)

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: function.php (hunk 4 above)
Comment (jasonrees1) [reply function.php]: Filename has been changed

File: function.php (hunk 1 above)
Comment (jasonrees1) [reply function.php]: All removed

File: index.php (hunk 2 above)
Comment (jasonrees1) [reply index.php]: Removed

File: index.php (hunk 5 above)
Comment (jasonrees1) [reply index.php]: required attribute added to all fields

File: index.php (hunk 3 above)
Comment (jasonrees1) [reply index.php]: All comments removed

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**